mm/routines/product.py

```diff
@@ -20,15 +20,18 @@ def get_configurable_associated_products(self, store_id=None, is_active=None, is
         legal_inputs = set([None, True, False])
         message = "Not a valid input value for '{0}'. Use: {1}"
 
-        assert type(store_id) is int or store_id is None, message.format('store_id', 'None or int')
-        assert is_active in legal_inputs, message.format('is_active', 'None, True or False')
-        assert is_visible in legal_inputs, message.format('is_visible', 'None, True or False')
+        assert type(store_id) is int or store_id is None, \
+            message.format('store_id', 'None or int')
+        assert is_active is True or is_active is False or is_active is None, \
```

Comment:
I sent an email with references for this. It's important you understand what I'm trying to explain. It will break you in other Python implementations.